lightwood/analysis/helpers/acc_stats.py

```diff
@@ -23,6 +24,7 @@ def analyze(self, info: Dict[str, object], **kwargs) -> Dict[str, object]:
         info['score_dict'] = evaluate_accuracy(ns.data, ns.normal_predictions['prediction'],
                                                ns.target, ns.accuracy_functions, ts_analysis={'tss': ns.tss})
         info['normal_accuracy'] = np.mean(list(info['score_dict'].values()))
+        log.info(f'Validation accuracy: {info["normal_accuracy"]}')
```

Comment:
I'd rather not log this, as stated before, I think users will just find an accuracy number confusing, doubly so a mean of all the scores, if you want to log something log the full dict with a format like this:
```
slog = '\n'.join([f'{k} -> round({v}, 3)' for k, v in info['score_dict'].items()])
log.info('Accuracy scores on validation (held out) data:\n {slog}')
```
... or some such, that's just my proposal